src/core/ComponentCell/style.ts

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+import { CommonThemeProps, getSpaces } from "../styles";
+
+const contentPositionMapping = {
+  center: "center",
+  left: "flex-start",
+  right: "flex-end",
+};
+export interface ComponentCellExtraProps extends CommonThemeProps {
+  contentPosition?: "left" | "center" | "right";
+}
+
+export const StyledTableData = styled("td")`
+  ${(props: ComponentCellExtraProps) => {
+    const spacings = getSpaces(props);
+    return `
+        align-items: center;
+        border: dashed 1px #ddd;
```

Comment:
Can you remove this border? Borders are just for the preview. I removed borders from Basic and Header cells as well.

Reply:
hmmm when I do this you basically can only see the react children, ie default uses a checkbox and it just looks like a checkbox. It seems like it might be a good idea to keep it for this case, maybe we can ask